plasma_cash/client/child_chain_service.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+
+class ChildChainService(object):
+
+    def __init__(self, base_url, verify=False, timeout=5):
+        self.base_url = base_url
+        self.verify = verify
+        self.timeout = timeout
+
+    def request(self, end_point, method, params=None, data=None, headers=None):
+        url = self.base_url + end_point
+        try:
+            response = requests.request(
+                           method=method,
+                           url=url,
+                           params=params,
+                           data=data,
+                           headers=headers,
+                           verify=self.verify,
+                           timeout=self.timeout,
+                       )
+        except requests.exceptions.Timeout:
+            raise Exception
+        except requests.exceptions.ConnectionError:
+            raise Exception
+
+        if response.ok:
+            return response
+        else:
+            raise Exception
```

Comment:
would recommend to throw custom exception and put the response (message?) into Exception message to show what's happening here.